src/z3c/etestbrowser/browser.py

```diff
@@ -79,7 +75,8 @@ def etree(self):
             match = RE_CHARSET.match(content_type)
             if match is not None:
                 charset = match.groups()[0]
-                content = content.decode(charset)
+                if six.PY2:  # pragma: no cover
+                    content = content.decode(charset)  # pragma: no cover
```

Comment:
I understand that this helps to avoid a coverage combine in `tox.ini` but it shadows the lines in coveralls, where it should be combined properly and declared as covered.

Reply:
https://pypi.org/project/coverage-python-version/ could be helpful here

Reply:
I'll give it a try.